cms/templates/cms/benefits_page.html

```diff
@@ -49,6 +49,17 @@ <h1 class="title">
         {% endblock %}
         </div>
       </div>
+      {% if '/organizations/' in request.path %}
+        <div class="mdl-cell mdl-cell--4-col">
```

Comment:
Can we add a sticky-top class here, like the one used in xPRO?

https://github.com/mitodl/mitxpro/blob/1d558b81d35787b6702de8e42e7fa362e0e58230/cms/templates/partials/subnav.html#L1

Reply:
It's not feasible as it's bootstrap 4 class but we are using bootstrap 3 here. Let me see what could be the alternatives.